browser/components/MarkdownPreview.js

```diff
@@ -25,6 +27,7 @@ function decodeHTMLEntities (text) {
 const { remote } = require('electron')
 const { app } = remote
 const path = require('path')
+const dialog = remote.require('electron').dialog
```

Comment:
OMG, I don't have to require twice :fearful:. However it cannot operate correctly because I want to use `dialog` in the renderer process. Thus I change it to `const dialog = remote.dialog`
> The Dialog is opened from Electron's main thread. If you want to use the dialog object from a renderer process, remember to access it using the remote:
> ```
>const {dialog} = require('electron').remote
>console.log(dialog)
>```
refs: https://github.com/electron/electron/blob/master/docs/api/dialog.md